x/ratelimit/client/cli/cli_test.go

```diff
@@ -0,0 +1,318 @@
+package cli_test
+
+import (
+	"fmt"
+	"testing"
+
+	"github.com/cosmos/cosmos-sdk/client/flags"
+	sdk "github.com/cosmos/cosmos-sdk/types"
+
+	clitestutil "github.com/cosmos/cosmos-sdk/testutil/cli"
+
+	strideclitestutil "github.com/Stride-Labs/stride/v4/testutil/cli"
+
+	"github.com/Stride-Labs/stride/v4/testutil/network"
+
+	"github.com/stretchr/testify/suite"
+
+	tmcli "github.com/tendermint/tendermint/libs/cli"
+
+	"github.com/Stride-Labs/stride/v4/x/ratelimit/client/cli"
+
+	"github.com/Stride-Labs/stride/v4/app"
+	"github.com/Stride-Labs/stride/v4/x/ratelimit/types"
+)
+
+type IntegrationTestSuite struct {
+	suite.Suite
+
+	cfg     network.Config
+	network *network.Network
+}
+
+func (s *IntegrationTestSuite) SetupSuite() {
```

Comment:
Is there a way to leverage the `app/test_helper` setup for these tests instead? Similar to what we do for the keeper tests

Reply:
I see, that makes sense. I'll reach out to you offline to talk this one out